### PR TITLE
DAS-1762 - Add Harmony Regridding Service to service.yml in UAT.

### PR DESCRIPTION
**Pull request by owenlittlejohns**
## Jira Issue ID

DAS-1762

## Description

This PR adds the basic plumbing for the new Harmony Regridding Service.

## Local Test Steps

* Pull this branch of the Harmony repository.
* Pull the latest `main` branch of [the Harmony Regridding Service repository](https://git.earthdata.nasa.gov/projects/SITC/repos/harmony-regridding-service/browse).
* Build the Harmony Regridding Service Docker image using the `bin/build-image` script in that repository.
* Update you local `.env` file to include `harmony-regridder` in the comma-separated list of services in `LOCALLY_DEPLOYED_SERVICES`.
* Re-bootstrap your local Harmony instance. You should see that it creates the worker and manager Docker containers for the Harmony Regridding Service.
* Issue the request below - it should give you the specified error (a validation error, for a non-geographic CRS):

Request:

```
http://localhost:3000/C1245618475-EEDTEST/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?forceAsync=true&maxResults=1&outputCrs=EPSG%3A6933
```

Expected exception:

```
Target CRS not supported: \"+proj=cea +lat_ts=30 +lon_0=0 +x_0=0 +y_0=0 +datum=WGS84 +units=m +no_defs\"
```

## PR Acceptance Checklist
* [x] Acceptance criteria met
* [x] Tests added/updated (if needed) and passing
* ~~Documentation updated (if needed)~~

### CODE REVIEW

File: config/services.yml [hunk 1]
```diff
@@ -666,6 +666,32 @@ https://cmr.uat.earthdata.nasa.gov:
...
+    data_operation_version: '0.17.0'
+    type:
+      <<: *default-turbo-config
+      params:
+        <<: *default-turbo-params
+        env:
+          <<: *default-turbo-env
+          STAGING_PATH: public/sds/harmony-regridder
+    umm_s:
+      - S1256563664-EEDTEST
```
Comment (flamingbear) [inline config/services.yml:680]: I couldn't find this service in the UAT MMT. Should I be able to?

**Review by flamingbear — APPROVED** [review]
I was able to run all of the testing steps and got the expected validation error.  Just a question about the UMMS record and whether that should be visible yet.

File: config/services.yml (hunk 1 above)
Comment (owenlittlejohns) [reply config/services.yml:680]: Discussed in Slack:

Matt used the service concept ID in the search bar in MMT. This did not retrieve the UMM-S record. (Nor for me)
Searching MMT for the service name "Harmony Regridding Service" was successful, though.

[Here is the URL for the service in UAT CMR](https://mmt.uat.earthdata.nasa.gov/services/S1256563664-EEDTEST). Alternatively, [here is the URL for the CMR search response for this UMM-S record](https://cmr.uat.earthdata.nasa.gov/search/services.umm_json?concept_id=S1256563664-EEDTEST).

**Review by chris-durbin — APPROVED** [review]
Successfully tested in a sandbox environment.

**Pull request merged**